ert3/console/_console.py

```diff
@@ -205,6 +211,20 @@ def _clean(workspace: Path, args: Any) -> None:
     ert3.console.clean(workspace, args.experiment_names, args.all)
 
 
+def _service(args: Any) -> None:
+    print("Checking status for storage service")
```

Comment:
Printing should go away...

Reply:
Should we get no feedback when running the checks? or should we use the logging module?

Reply:
I think we could just start out with it being completely silent on success :)